rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly
```

Comment:
I noticed that the nightly toolchain is used but it's always separately specified, so I added it here so it could be picked up by the tooling automatically.